src/m365/spo/commands/user/user-remove.ts

```diff
@@ -119,10 +88,41 @@ class SpoUserRemoveCommand extends SpoCommand {
       });
 
       if (result.continue) {
-        await removeUser();
+        await this.removeUser(logger, args);
       }
     }
   }
+
+  private async removeUser(logger: Logger, args: CommandArgs): Promise<void> {
+    if (this.verbose) {
+      logger.logToStderr(`Removing user from  subsite ${args.options.webUrl} ...`);
+    }
+
+    let requestUrl: string = '';
+
+    if (args.options.id) {
+      requestUrl = `${encodeURI(args.options.webUrl)}/_api/web/siteusers/removebyid(${args.options.id})`;
+    }
+
+    if (args.options.loginName) {
+      requestUrl = `${encodeURI(args.options.webUrl)}/_api/web/siteusers/removeByLoginName('${formatting.encodeQueryParameter(args.options.loginName as string)}')`;
+    }
```

Comment:
let's make it a bit nicer. 

- Let's remove some not needed empty lines. 
- Start with initializing what is the common part for both endpoints
- use else as it is either this or that not possible to have both as it is part of an optionset

```suggestion
    let requestUrl: string = `${encodeURI(args.options.webUrl)}/_api/web/siteusers/`;
    if (args.options.id) {
      requestUrl += `removebyid(${args.options.id})`;
    }
   else if (args.options.loginName) {
      requestUrl += `removeByLoginName('${formatting.encodeQueryParameter(args.options.loginName as string)}')`;
    }
```